### PR TITLE
TASK-65: Identified Trash Frontend

### LINKED ISSUE #229: TASK-65: Identified Trash Frontend
**Task Description**

As a developer, I want a page to be made to show extra information about a type of waste so that #34 can be completed.

**Acceptance Criteria**

- [ ] Documentation present where necessary
- [ ] Unit and UI tests are implemented for every new component and function
- [ ] The application should still work with seeded data
- [ ] Must match [Mockup](https://www.figma.com/file/FJPXMYI4ULCnuCEZvPs67k/Mull?node-id=2047%3A76571)

**Note**: Mock info is fine. No need to add analysis breakdown.
TODO: Update Figma

### PR DESCRIPTION
**Pull request by cindyslittleplanet**
This PR closes #229 .

You can access the modal by going to http://localhost:4200/camera and then clicking on the `CLICK HERE` button. 
As of now, all the details appearing on the modal are hard coded and will be changed in future tickets.

### CODE REVIEW

File: apps/mull-ui/src/app/pages/waste-recognition-page/identified-waste-modal.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import { faTimes } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Dialog } from '@material-ui/core';
+import React from 'react';
```
Comment (JimLau49) [inline apps/mull-ui/src/app/pages/waste-recognition-page/identified-waste-modal.tsx]:
```suggestion
import React, {useState} from 'react';
```

File: apps/mull-ui/src/app/pages/waste-recognition-page/identified-waste-modal.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  wasteTitle,
+  wastePicture,
+  wasteInfo,
+}: IdentifiedWasteModalProps) => {
+  wasteIcon = '../../.' + svgMap[1];
+  wasteTitle = 'This is recyclable!';
+  wastePicture = 'http://pm1.narvii.com/5951/2922186bdf76edeb36250994eb99f1c32f31aea9_00.jpg';
+  wasteInfo = 'Needs to be cleaned before recycling. Otherwise it goes to trashbin.';
+
+  const [open, setOpen] = React.useState(false);
```
Comment (JimLau49) [inline apps/mull-ui/src/app/pages/waste-recognition-page/identified-waste-modal.tsx]:
```suggestion
  const [open, setOpen] = useState(false);
```

**Review by JimLau49 — APPROVED** [review]
Good work on the modal 🚀 . Left a small comment as well as tried some methods to try to get rid the the `!important` in some of the styles but I'm not too familiar with Material UI and its not  a big enough issue to block it 

**Acceptance Criteria**

- [x] Documentation present where necessary
- [x] Unit and UI tests are implemented for every new component and function
- [x] The application should still work with seeded data
- [x] Must match [Mockup](https://www.figma.com/file/FJPXMYI4ULCnuCEZvPs67k/Mull?node-id=2047%3A76571)

**Review by ys-lin — APPROVED** [review]
LGTM :D 
**Acceptance Criteria**

- [x] Documentation present where necessary
- [x] Unit and UI tests are implemented for every new component and function
- [x] The application should still work with seeded data
- [x] Must match [Mockup](https://www.figma.com/file/FJPXMYI4ULCnuCEZvPs67k/Mull?node-id=2047%3A76571)
image

File: apps/mull-ui/src/app/pages/waste-recognition-page/identified-waste-modal.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  const handleOpen = () => {
+    setOpen(true);
+  };
+  const handleClose = () => {
+    setOpen(false);
+  };
```
Comment (cristian-aldea) [inline apps/mull-ui/src/app/pages/waste-recognition-page/identified-waste-modal.tsx]: 💡 Honestly, if your methods are only calling setOpen, you might as well remove them and call setOpen directly in your onClicks

**Review by cristian-aldea — APPROVED** [review]
Had one small 💡 suggestion, but otherwise, everything looks great!

- [x] Documentation present where necessary
- [x] Unit and UI tests are implemented for every new component and function
- [x] The application should still work with seeded data
- [x] Must match [Mockup](https://www.figma.com/file/FJPXMYI4ULCnuCEZvPs67k/Mull?node-id=2047%3A76571)

**Pull request merged**